consent/strategy_default_test.go

```diff
@@ -463,6 +465,26 @@ func runLogout(t *testing.T, method string) {
 			jar:              newValidAuthCookieJar(t, reg, logoutServer.URL, "logout-session-temp6", "logout-subject-temp6"),
 			expectBody:       "redirected to default server1234custom",
 		},
+		{
+			d:   "should not append a state param if no state was passed to logout server",
+			lph: acceptLogoutChallenge(apiClient, "temp7"),
+			params: url.Values{
+				"post_logout_redirect_uri": {defaultRedirServer.URL + "/custom"},
+				"id_token_hint": {genIDToken(t, reg, jwtgo.MapClaims{
+					"aud": defaultClient.OutfacingID,
+					"iss": conf.IssuerURL().String(),
+					"sub": "logout-subject-temp7",
+					"sid": "logout-session-temp7",
+					"exp": time.Now().Add(-time.Hour).Unix(),
+					"iat": time.Now().Add(-time.Hour * 2).Unix(),
+				})},
+			},
+			expectStatusCode: http.StatusOK,
+			jar:              newValidAuthCookieJar(t, reg, logoutServer.URL, "logout-session-temp7", "logout-subject-temp7"),
+			subject:          "logout-subject-7",
+			expectBody:       "redirected to default servercustom",
+			expectRequestURI: "/custom",
+		},
```

Comment:
An alternative would be to remove the `state` value from `expectBody` and instead add the `expectRequestURI` to all test-cases.